src/web/gwc/src/main/resources/GeoServerApplication.properties

```diff
@@ -7,6 +7,8 @@ GWC.diskQuotaLoadFailed = Loading quota store failed, the disk quota subsystem i
 GWCSettingsPage.description                = Configure the global settings for the embedded GeoWebCache
 GWCSettingsPage.enableWMSIntegration       = Enable direct integration with GeoServer WMS
 GWCSettingsPage.enableWMSIntegration.title = Allows the regular WMS to serve cached content. When this option is enabled, use tiled=true for GetCapabilities in the GeoServer WMS to include the WMS-C VendorSpecificCapabilities
+GWCSettingsPage.requireTiledParameter      = Explicitly require TILED Parameter
+GWCSettingsPage.requireTiledParameter.title= When set to true, explicitly require TILED Parameter, otherwise accept all requests for tiling
```

Comment:
```suggestion
GWCSettingsPage.requireTiledParameter.title= When set to true, explicitly require TILED Parameter, otherwise consider all requests for tile caching
```